guide/russian/cplusplus/index.md

```diff
@@ -78,8 +79,8 @@ int main()
 *   Не забудьте сообщить компьютеру, что это конец функции main (). Для этого вы добавляете закрывающуюся фигурную скобку «}». Вы столкнетесь с ошибкой перед выполнением программы, если вы не включите **}** .
 
 ### Код должен выглядеть примерно так:
-
-![Img](https://i.imgur.com/d1liGwI.png)
+https://cdn-media-1.freecodecamp.org/imgr/
```

Comment:
Is this line needed?
```suggestion
```

Reply:
@RandellDawson how about replacing 
`![Img](https://i.imgur.com/d1liGwI.png)`
with
`![Img](https://cdn-media-1.freecodecamp.org/imgr/d1liGwI.png)`
instead of removing
`https://cdn-media-1.freecodecamp.org/imgr/`